src/Pages/Profile.js

```diff
@@ -0,0 +1,86 @@
+import { useState, useEffect, useContext } from 'react'
+import Card from '../components/UI/Card';
+import { AuthContext } from '../store/auth-context';
+
+const Profile = () => {
+  const [loading, setLoading] = useState(true);
+  const [username, setUsername] = useState(null);
+  const [website, setWebsite] = useState(null);
```

Comment:
Controlled inputy - to taťka může :+1: . Ale ta výchozí hodnota by asi měla být prázdný string, než null. Hodnota text inputů může být jenom string.